src/main/java/uk/gov/hmcts/reform/sscs/domain/wrapper/Decision.java

```diff
@@ -0,0 +1,125 @@
+package uk.gov.hmcts.reform.sscs.domain.wrapper;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.swagger.annotations.ApiModelProperty;
+import java.util.Objects;
+
+@JsonInclude(value = JsonInclude.Include.NON_EMPTY)
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class Decision {
```

Comment:
Decisions are part of COH so not needed for Tribunals API I don't think?

Reply:
OnlineHearing data object is used by both MYA and COR, can't removed it unless if we modify the frontend